js/app/page.tsx

```diff
@@ -274,22 +286,24 @@ function NavBar() {
     <Tabs index={tabIndex}>
       <TabList>
         <Box flex="1" display="flex">
-          {tabs.map(({ name, href, badge }) => {
+          {tabs.map(({ name, href, badge, disable }) => {
             return (
               <Tab
                 key={name}
                 onClick={() => {
                   setLocation(href);
                 }}
+                isDisabled={isLoading || isFlagLoading || disable}
+                hidden={disable}
               >
                 {name}
                 {badge}
               </Tab>
             );
           })}
         </Box>
-        {!isLoading && !isDemoSite && <Filename />}
-        {!isLoading && (
+        {!isLoading && !isFlagLoading && !isDemoSite && <Filename />}
```

Comment:
I'm not sure do you need `isFlagLoading` here?

Reply:
You are right! I think we don't need to check `isFlagLoading` here.